build.config

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 CAKE_VERSION=0.37.0
 DOTNET_VERSION=3.1.102
-VERSION_PREFIX=0.9.3
```

Comment:
why do u remove build version?

Reply:
Build version is added in this PR.And it is set in environment variable instead of config.